src/luos.c

```diff
@@ -1069,3 +1095,18 @@ void Luos_Run(void)
             break;
     }
 }
+
+void Luos_DetectionAsk(service_t *service, uint16_t node_id)
```

Comment:
I don't see how this can work.
Nobody have any ID here!
Users need to put a 0?
If yes I think we should hide the node_id to have something like  `void Luos_DetectionAsk(service_t *service)`
 and put the 0 by ourself
.If users want to request detection they just can make a message for it (not sure about it).

Reply:
1. Is it ok for u if we dont pass the node_id but we call Robus_GetNode() into Luos api in order to get the node_id of the current node?
2. the users can indeed request a detection by creating their own message. In that way the can always demand to the node 1 to launch the detection and maybe this can work for the projects that we need to know the topology of the network

Reply:
@BiliouriV what is the state of this one?